src/main/webapp/js/pages/index.js

```diff
@@ -21,7 +21,7 @@ window.addEventListener('load', () => {
 });
 
 document.querySelector('.course-list').addEventListener('click', () => {
-  Calendar.clear();
+  const MAX_PAGINATION_SCHEDULES = 15;
```

Comment:
typically these constant declarations should go at the top of the file, outside of any function (line 17).